packages/ui-editable/src/Editable/props.ts

```diff
@@ -71,8 +71,10 @@ type EditableOwnProps = {
    */
   mode: 'view' | 'edit'
   /**
-   * Called when the component's mode changes
-   * @param {string} new_mode
+   * Called when the component's mode changes.
+   *
+   * Parameter:
+   * - newMode: string
```

Comment:
I think you can remove the JsDoc of the parameter here, because the TS type shows it well enough, that it has a string `newMode` param:
_____________________________________
![image](https://user-images.githubusercontent.com/11446033/174297621-ddd6ecb8-2859-46fc-85b4-9fd3b8cd47e3.png)
_____________________________________